Doc/library/decimal.rst

```diff
@@ -1033,10 +1033,10 @@ New contexts can also be created using the :class:`Context` constructor
 described below. In addition, the module provides three pre-made contexts:
 
 
-.. class:: BasicContext
+.. data:: BasicContext
 
-   This is a standard context defined by the General Decimal Arithmetic
-   Specification.  Precision is set to nine.  Rounding is set to
+   This is a predefined context object defined by the General Decimal Arithmetic Specification.
```

Comment:
The term "standard" is the correct term. Please do not change this as this is unrelated to the issue.